PokeAlarm/Manager.py

```diff
@@ -242,7 +242,18 @@ def handle_pokemon(self, pkmn):
                 return
         else:
             log.debug("Pokemon inside geofences was not checked because no geofences were set.")
-
+        
+        height = pkmn['height']
```

Comment:
Can we bring these into one line:
`height, weight, gender = pkmn['height'], pkmn['weight'], pkmn['gender']`